### PR TITLE
Made rollbar integration optional

### PR DESCRIPTION
**Pull request by sahil-khanna-instacart**
## What
Added an option to disable roll bar

## Why
We want to use different error reporting system. Having a flexibility to disable roll bar explicitly would be useful.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,2 @@
--e .
+setuptools==48.0.0
```
Comment (sahil-khanna-instacart) [inline requirements.txt:1]: Pinning to older version because new version does not support `.` in requirements.txt

Comment (montanalow) [reply requirements.txt:1]: Can you update https://github.com/instacart/lore/blob/master/lore/dependencies.py if this is a real dependency?

File: lore/util.py [hunk 2]
```diff
@@ -299,15 +299,16 @@ def calling_logger(height=1):
...
-            import rollbar
-            rollbar.init(
-                os.environ.get("ROLLBAR_ACCESS_TOKEN", None),
-                allow_logging_basic_config=False,
-                environment=env.NAME,
-                enabled=(env.NAME != env.DEVELOPMENT),
-                handler='blocking',
-                locals={"enabled": True}
-            )
+            if not getattr(sys, 'lore_disable_rollbar', False):
```
Comment (montanalow) [inline lore/util.py:302]: I would make this a check for whether or not rollbar is present, instead of having to set a global on sys.

**Review by montanalow — CHANGES_REQUESTED** [review]

File: lore/util.py (hunk 2 above)
Comment (sahil-khanna-instacart) [reply lore/util.py:302]: Do you mean to check whether `rollbar` is installed?

**Pull request closed without merging**